broker/cmd/dejaqcli-producer/main.go

```diff
@@ -165,6 +165,11 @@ func main() {
 		logger.WithError(err).Fatal("cannot handshake")
 	}
 
+	strategy := "StrategySingleBurst"
+	if c.strategy == sync_produce.StrategyConstantBursts {
+		strategy = "StrategyConstantBursts"
+	}
+	logger.Info(fmt.Sprintf("strategy: %s", strategy))
```

Comment:
why do you need a new variable and a new string? 
logger.Infof("starting with strategy: ", c.strategy)